lib/overcommit/hook/shared/rake_target.rb

```diff
@@ -0,0 +1,24 @@
+module Overcommit::Hook::Shared
+  # runs specified rake targets. It fails on the first non-
+  # successfull exit.
+  #
+  module RakeTarget
+    def run(targets = config['targets'])
+      if targets.nil?
+        raise "RakeTarget: targets parameter is nil. Add at least one task to \
+          the targets parameter. Valid: Array of target names"
+      elsif !targets.is_a?(Array)
+        raise "RakeTarget: targets parameter is not an array. Valid: Array of \
+          rake target names"
+      end
```

Comment:
You could simplify this into a single case by `Array`ifying `targets`:

```ruby
if Array(targets).empty?
  raise ...
end
```

This also allows users to specify a single string for `targets` so they don't need to explicitly make it an array, making it slightly more user-friendly. Overcommit takes this approach with other configuration options where it makes sense.